pkg/interface/contacts/src/js/components/lib/contact-card.js

```diff
@@ -489,35 +489,36 @@ export class ContactCard extends Component {
     //TODO "Share card" if it's /me -> sends to /~/default of recipient
     return (
       <div className="w-100 h-100 overflow-hidden">
-        <div className="w-100 bg-white bb b--gray4">
+        <div className="flex justify-between w-100 bg-white bb b--gray4 ">
```

Comment:
? We're centring these in the middle now?

Reply:
No, this is so that the button groups are flush left or right

Reply:
It makes more sense to have the remove button separated from the primary actions.

Reply:
Yeah I pointed this out in a github issue or asana task last week — it was way, way too easy to delete remove oneself/another person from a group